src/app/shared/navigation/navigation.component.html

```diff
@@ -2,31 +2,35 @@
   <ul class="navbar-nav">
     <li class="nav-item">
       <a
-        class="nav-link"
+        class="nav-link item"
         routerLink="/home"
         routerLinkActive="active"
-        class="item"
         [routerLinkActiveOptions]="{ exact: true }"
         >Home</a
       >
     </li>
     <li class="nav-item">
-      <a
-        class="nav-link"
-        routerLink="/products"
-        routerLinkActive="active"
-        class="item"
+      <a class="nav-link item" routerLink="/products" routerLinkActive="active"
         >Products</a
       >
     </li>
     <li class="nav-item">
-      <a
-        class="nav-link"
-        routerLink="/contact"
-        routerLinkActive="active"
-        class="item"
+      <a class="nav-link item" routerLink="/contact" routerLinkActive="active"
         >Contact</a
       >
     </li>
+    <li class="nav-item" *ngIf="this.isLoggedIn">
```

Comment:
To jest ok ale tak na przyszlosc - poczytaj o dyrektywach strukturalnych (i jak dziala ngIf) i mozesz zastapic ne ngIfy i wstrzykiwanie w komponentach authServicu jedna dyrektywa na elemencie ktory bedziesz chcial pokazywac tylko zalogowanym :)

Reply:
ok